core/bug_api.php

```diff
@@ -1545,6 +1550,93 @@ function bug_get_newest_bugnote_timestamp( $p_bug_id ) {
 }
 
 /**
+ * For a list of bug ids, return array of bugnote stats:
```

Comment:
phpDoc block should start with a 1-line summary of what the function does, followed by a period or empty line.